node/src/chain_spec.rs

```diff
@@ -88,21 +114,353 @@ pub fn mangata_inflation_config() -> InflationInfo<Balance> {
 	}
 }
 
+pub fn kusama_mainnet_config() -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "MGX".into());
+	properties.insert("tokenDecimals".into(), 18.into());
+	properties.insert("ss58Format".into(), 42.into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"Mangata Kusama Mainnet",
+		// ID
+		"mangata_kusama_mainnet",
+		ChainType::Development,
+		move || {
+			mangata_genesis(
+				// initial collators.
+				vec![
+					(
+						kusama_mainnet_keys::ALICE_SR25519
+							.parse::<sr25519::Public>()
+							.unwrap()
+							.into(),
+						kusama_mainnet_keys::ALICE_SR25519
+							.parse::<sr25519::Public>()
+							.unwrap()
+							.into(),
+					),
+					(
+						kusama_mainnet_keys::BOB_SR25519.parse::<sr25519::Public>().unwrap().into(),
+						kusama_mainnet_keys::BOB_SR25519.parse::<sr25519::Public>().unwrap().into(),
+					),
+				],
+				// Initial relay account
+				kusama_mainnet_keys::RELAY_SR25519.parse::<sr25519::Public>().unwrap().into(),
+				// Sudo account
+				kusama_mainnet_keys::SUDO_SR25519.parse::<sr25519::Public>().unwrap().into(),
```

Comment:
// offtop, 

do we want sudo account on kusama ? who will be in charge such account? Doesnt it imply that someone could mint MGA ?

Reply:
Yes, right now we do want to have it, and with one of the first upgrades we will get rid of it.